src/DotNetBridge/Entrypoints.cs

```diff
@@ -87,5 +87,59 @@ public static CommonOutputs.TransformOutput CreateVariableColumn(IHostEnvironmen
             var xf = VariableColumnTransform.Create(env, inputOptions, inputOptions.Data);
             return new CommonOutputs.TransformOutput { Model = new TransformModelImpl(env, xf, inputOptions.Data), OutputData = xf };
         }
+
+        public sealed class Input
+        {
+            [Argument(ArgumentType.Required, HelpText = "The dataset to be scored", SortOrder = 1)]
+            public IDataView Data;
+
+            [Argument(ArgumentType.Required, HelpText = "The predictor model to apply to data", SortOrder = 2)]
+            public PredictorModel PredictorModel;
+
+            [Argument(ArgumentType.AtMostOnce, HelpText = "Suffix to append to the score columns", SortOrder = 3)]
+            public string Suffix;
+        }
+
+        public sealed class Output
```

Comment:
Is this name to general since it exists in a file that contains more than one type of Input/Output class?